libs/langchain-google-genai/src/chat_models.ts

```diff
@@ -589,7 +589,7 @@ export class ChatGoogleGenerativeAI
       this.model;
     this.model = this.modelName;
 
-    this.convertSystemMessageToHumanContent = this.useSystemInstructions();
+    this.convertSystemMessageToHumanContent = this.computeUseSystemInstruction();
```

Comment:
Suggestion: remove this line completely since in google common convertSystemMessageToHumanContent is never actually set in the constructor. Instead in every reference of this.convertSystemMessageToHumanContent like in the invocation of convertBaseMessagesToContent, call this.useSystemInstruction() instead